bin/github-label-sync.js

```diff
@@ -137,8 +137,11 @@ function merge(files) {
 		});
 	});
 
+	// Make sure leading `#` are stripped from the color property.
 	Object.keys(data).forEach((key) => {
-		labels.push(data[key]);
+		const label = data[key];
+		label.color = label.color.replace('#', '');
```

Comment:
This removes the first instance of `#`, which is differently from removing it from the start of the string.

For example: `'a#b'.replace('#', '')` would return `'ab'`.

Should we change the code to only remove it from the start of the string?
```suggestion
		if (label.color.startsWith('#')) {
			label.color = label.color.replace('#', '');
		}
```

Reply:
That's what I wrote initially, but then my inner "tester voice" said fewer branches are better LOL! My logic was that no existing colors have the `#` character, so the conditional is unnecessary.

It's trivial either way, I 'll update it as you prefer.

Reply:
I added the conditional logic and also convert `color` to a string. I came across a problem with hexadecimal colors like `888888`. Previously, it didn't matter if the colors were a string or number.